lib/src/platform_client.dart

```diff
@@ -62,6 +64,22 @@ class PlatformClient {
     await _httpPost('/api/smartapp/verify', body);
   }
 
+  /// Confirms a verification code that was sent to a phone number by [sendPhoneVerificationCode].
+  ///
+  /// [phoneNumber] is the phone number in E.164 format and [verificationCode] is the verification code. If successful,
+  /// the returned [Credentials] can be used to [createAccount] (if [Credentials.isNewUser] is `true`) or to
+  /// [loginWithCredentials].
+  Future<Credentials> confirmPhoneVerificationCode(String phoneNumber, String verificationCode) async {
+    String body = jsonEncode({
+      'authCode': verificationCode,
+      'phoneNumber': phoneNumber,
+    });
+
+    Map<String, dynamic> response = await _httpPost('/api/smartapp/verify/confirm', body);
```

Comment:
It would be nice to have this URL as: `api/smartapp/verify/phone/confirm' or either merge those two into one endpoint. I understand though that this would be a breaking change as I haven'e see any change in platform concerning endpoint URL.

Another one for Later!?!

Reply:
Yep, another one for later. I think we'll need an authentication service and will get the opportunity to redesign the APIs.